abe/helper_functions/email_helpers.py

```diff
@@ -100,28 +103,30 @@ def ical_to_dict(cal):
     event_def['description'] = desc
     return event_def, sender
 
+
 def get_messages_from_email():
     """ Fetches unread emails from the email address
     specified by the environmental variable ABE_EMAIL
-    (password given by env var ABE_PASS). Returns a 
+    (password given by env var ABE_PASS). Returns a
     list of messages.
 
     :return: List of email message objects
     """
     pop_conn = poplib.POP3_SSL('pop.gmail.com')
-    pop_conn.user(os.environ['ABE_EMAIL'])
-    pop_conn.pass_(os.environ['ABE_PASS'])
-    pop3info = pop_conn.stat() #access mailbox status
+    pop_conn.user(ABE_EMAIL)
+    pop_conn.pass_(ABE_PASS)
+    pop3info = pop_conn.stat()  # access mailbox status
```

Comment:
`pop3info` is ignored. Is the call to `pop_conn.stat()` for effect, or should it be removed?

Reply:
This can be removed